cgi-bin/LJ/Console/Command/Finduser.pm

```diff
@@ -77,6 +93,13 @@ sub execute {
     } elsif ($crit eq 'user') {
         $data = LJ::canonical_username($data);
         $userlist = $dbh->selectcol_arrayref("SELECT userid FROM user WHERE user = ?", undef, $data);
+    } elsif ( $crit eq 'openid' ) {
+        # Use wildcard search to match either HTTP or HTTPS entries.
+        my $rows = $dbh->selectcol_arrayref(
+            "SELECT userid, identity FROM identitymap WHERE idtype='O' AND identity LIKE ?",
+            { Columns => [1,2] }, "%$data" );
```

Comment:
I had never seen this pattern before (or had forgotten it) and looked up what it does. I vaguely feel like this would have saved me so much time...